### DISCUSSION TITLE
Many Thanks!

### DISCUSSION THREAD

**Discussion by ColonelBlimp** [musescore/MuseScore#16670] [General]
Just a thank you to all involved in developing and supporting this project.

What a great piece of software! After struggling with Sibelius, coming to MuseScore was a delight.

Many thanks!

**Comment by smj-edison** [comment]
Figured I shouldn't start my own thread, but I also want to chip in and say thank you to all involved! MuseScore 4 has been an absolute delight to work with, even with its rough edges. Muse Sounds makes playback gorgeous, and it's much more intuitive.
